tools/ci/tc/decision.py

```diff
@@ -101,14 +100,13 @@ def get_extra_jobs(event):
     return jobs
 
 
-def filter_jobs(jobs, event):
-
+def filter_tasks(tasks, event):
     # Exclude browser stability tests for exports from that specific browser.
     try:
         if event["pull_request"]["user"]["login"] == "chromium-wpt-export-bot":
```

Comment:
I guess the other thing we could do is add an `exclude-users` property in the yaml file and use that to do the filtering, like we do with `filter_schedule_if`.

Reply:
To be clear, I don't consider this review blocked on changing that; it seems reasonable to just land it with the hardcoded rules. But if you want to try adding it as a feature in the YAML file, you're welcome to do so; it is probably the more principled approach.